### PR TITLE
[do NOT merge]  Just a draft to run packit PRT

### PR DESCRIPTION
**Pull request by vsedmik**
### Problem Statement
Just a draft PR to test possible regressions for https://github.com/Katello/katello/pull/11058

### PRT test Cases example
This run should pass, we can start here and trigger more related stuff later.
```
trigger: test-robottelo
pytest: tests/foreman/api/test_contentview.py
Katello:
    katello: 11058
```

### CODE REVIEW

**Comment by vsedmik** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/api/test_contentview.py
Katello:
    katello: 11058
```

**Comment by vsedmik** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/cli/test_contentviewfilter.py tests/foreman/cli/test_docker.py
Katello:
    katello: 11058
```

**Comment by vsedmik** [comment]
```
=========== 3 failed, 138 passed, 316 warnings in 4150.27s (1:09:10) ===========
```
2x failure on known issue of AK update:
```
12:59:39  E           robottelo.exceptions.CLIReturnCodeError: CLIReturnCodeError(status=64, stderr="Could not update the activation key:\n  Missing arguments for '--name'.\n", msg='Command "activation-key update" finished with status 64\nstderr contains:\nCould not update the activation key:\n  Missing arguments for \'--name\'.\n'
```
1x assertion failure on `hammer product info` here https://github.com/SatelliteQE/robottelo/blob/88e5aa0400d0264fef820bbe09a04cb68173a649/tests/foreman/cli/test_docker.py#L176
looks like it did not return the expected repos for that product

**Comment by ianballou** [comment]
It looks like the failing docker repo test has a very similar one right after it in robottelo. If that later test is passing, perhaps the failure is a fluke.

**Comment by vsedmik** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/cli/test_contentviewfilter.py tests/foreman/cli/test_docker.py -k create_repos_using_same_product
Katello:
    katello: 11058
```

**Comment by vsedmik** [comment]
Or I wonder if repos creation failed silently and thus they weren't found the module-scoped product (where repos from other tests within the module were present).
Anyway, let's see how it runs standalone ^^

File: tests/foreman/api/test_contentview.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 
 """
 
+# This file fully passed in snaps 72-75, let's start here.
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_contentview.py:15]: Just for awareness, If you would like to run an upstream PRT, the PRT comment can be added directly to the Jenkins job while build with parameters, without requiring robottelo PRs, and you can also post results on any GH PRs where you wish, such as here on upstream Katello PR

**Comment by vsedmik** [comment]
Standalone it passed @ianballou ^ 
```
=== 1 passed, 140 deselected, 149 warnings in 1361.42s (0:22:41) ===
```

@Gauravtalreja1 @omkarkhatavkar Not sure why the PRT results comment is not visible here. The `curl -L -X POST` from CI was sent ok. Any idea?

**Comment by omkarkhatavkar** [comment]
> Standalone it passed @ianballou ^
> 
> ```
> === 1 passed, 140 deselected, 149 warnings in 1361.42s (0:22:41) ===
> ```
> 
> @Gauravtalreja1 @omkarkhatavkar Not sure why the PRT results comment is not visible here. The `curl -L -X POST` from CI was sent ok. Any idea?

PR is raised here https://github.com/SatelliteQE/robottelo/pull/16684

**Pull request closed without merging**